docs/python_docs/environment.yml

```diff
@@ -36,3 +36,4 @@ dependencies:
   - breathe
   - mock
   - awscli
+  - autodocsumm
```

Comment:
Is that from here? https://github.com/Chilipp/autodocsumm

Reply:
yeah that's where it's from.